ttn/handler.py

```diff
@@ -20,17 +20,22 @@ class HandlerClient:
     def __init__(self, app_id,
                  app_access_key,
                  discovery_address="discovery.thethings.network:1900",
-                 certificate=None):
+                 certificate=""):
         self.app_id = app_id
         self.app_access_key = app_access_key
         self.discovery_address = discovery_address
```

Comment:
If this is not used outside of the constructor, no need to store it - make it a parameter of `__open`